mtags-java/src/main/scala/scala/meta/internal/pc/JavaKeyword.scala

```diff
@@ -0,0 +1,35 @@
+package scala.meta.internal.pc
+
+case class JavaKeyword(
+    name: String,
+    level: JavaKeyword.Level
+)
+
+object JavaKeyword {
+  sealed trait Level
+
+  case object TopLevel extends Level
+  case object ClassLevel extends Level
+  case object MethodLevel extends Level
+
+  val topLevelKeywords: List[JavaKeyword] = List(
```

Comment:
I think that `record` may be missing from this list (and maybe from `classLevelKeywords` as well?). :slightly_smiling_face: